openqabot/approver.py

```diff
@@ -348,3 +362,18 @@ def osc_approve(inc: IncReq) -> bool:
             return False
 
         return True
+
+    def git_approve(self, inc: IncReq, msg: str) -> bool:
```

Comment:
Why not a static method here?

Reply:
If with `static` you mean a `class` method, I think is because of the `self.gitea_token` on line 370

Reply:
I just noticed that `osc_approve` is static, and I was wondering if it should keep the approach the same and not a blocker. Maybe we can look at it later but for now I will approve.